### ISSUE TITLE
Screen readers badly render links in "federated reply" block

### ISSUE THREAD

**Issue by talksina** [Automattic/wordpress-activitypub#875]
### Quick summary

@pfefferle when using the "reply to" bookmarklet, at beginning of a WordPress post, an unpleasant output for screen readers appears - for example "↬mastodon.uno/@uaar/113039993829553640"
It is the direct link at the post/conversation we are replying to and it's correct, but the "a href" HTML tag has no link text! So, for fediverse newbies it's not easy to figure it is a post's reply because the screen reader reads the alpha-numeric link and a user would never click something "anonymous"; it seems like a malicious code.

### Steps to reproduce

1. open any Mastodon/Fediverse post;
2. use the "reply from [blog]" bookmarklet so that it creates a new post;
3. the issue can be verified also in previewing the post after having saved it into draft.

### What you expected to happen

The "federated reply" block should look, in front-end, like this:
"this post is in reply to: [mentioned user] conversation: view original thread" - and text string in the block should be editable from within the block, so that a user can introduce it as they want (I mean the link text and "this post is in reply to"...).

### What actually happened

Now just an emoji and the post's alphanumeric link/code appear. Alone, with no explanation on what it is.

### Impact

All

### Available workarounds?

No but the platform is still usable

### Logs or notes

ActivityPub version 3.0.1, WordPress 6.6.1, Gutenberg 19.1. This issue has no workarounds and prevents me from replying to conversations. - Not even verified if clicking the link opens a new window. If it doesn't and takes me out of the blog, it's even more serious.

**Comment by talksina** [comment]
Found a temporary small workaround to respond to posts without it seeming to be a malicious code.

1. place the "federated response" block as first block for the post;
2. add a "read more" block right after it;
3. select the newly added "read more" block, and activate the "hide" setting on "read more"'s block settings.

It is quite an unpleasant workaround as it currently prevents me from writing other stuff in there, and it forces me to remember I have to write a mention to the original author and fediverse conversation's list, appended to the post. But it's all what I have now.
